bin/utest/vm_map.c

```diff
@@ -15,6 +15,7 @@ int test_sharing_memory_simple(void) {
   size_t pgsz = getpagesize();
   char *map =
     mmap(NULL, pgsz, PROT_READ | PROT_WRITE, MAP_ANON | MAP_SHARED, -1, 0);
+
   assert(map != (char *)MAP_FAILED);
```

Comment:
Please don't add empty lines in random pieces of code.

Reply:
Fixed.